code/modules/reagents/reagent_containers/food/snacks.dm

```diff
@@ -311,6 +314,18 @@
 		contents += W
 		return 1 //No afterattack here
 
+//For slipping solid junk inside food items, like hiding a PDA inside a loaf of bread or something.
+/obj/item/weapon/reagent_containers/food/snacks/proc/can_hold(obj/item/weapon/W)
+	if(storage_slots < 1)
+		return FALSE
+	if(W.w_class > W_CLASS_SMALL)
+		return FALSE
+	if(W.w_class < src.w_class)
```

Comment:
If it's *smaller* it fails?

Reply:
W is the item being placed, src is the fooditem

Reply:
Yes.
Read it again.

Reply:
Right, so if the `w_class` of the item being placed into the food is *smaller* than the `w_class` of the food item, it can't be inserted? How does that make sense?

Reply:
Oh right, my bad

Reply:
Will also fix in a couple hours